lib/jekyll/related_posts.rb

```diff
@@ -42,7 +42,7 @@ def build_index
     end
 
     def lsi_related_posts
-      self.class.lsi.find_related(post.content, 11) - [post]
+      self.class.lsi.find_related(post, 11)
```

Comment:
LSI should really be based on the content of the post, no? Doesn't this change the functionality of the indexer?